src/components/CookieNotice/CookieNotice.scss

```diff
@@ -0,0 +1,68 @@
+@import "constants/style.scss";
+
+$cookie-consent-width: 100%;
+$cookie-consent-top: 100%;
+$cookie-consent-position: sticky;
```

Comment:
- I think the position should be `fixed` here, since scrolling on the page wont change the position. Or I just don't get it
- Since you're only using this variable once in `.cookie-notice--show` you don't need the explicit variable here or won't change it